tfexec/internal/e2etest/util_test.go

```diff
@@ -18,29 +19,34 @@ import (
 )
 
 const testFixtureDir = "testdata"
+const masterRef = "refs/heads/master"
 
 func runTest(t *testing.T, fixtureName string, cb func(t *testing.T, tfVersion *version.Version, tf *tfexec.Terraform)) {
 	t.Helper()
-	runTestVersions(t, []string{
+
+	versions := []string{
 		testutil.Latest011,
 		testutil.Latest012,
 		testutil.Latest013,
-	}, fixtureName, cb)
+	}
+	if override := os.Getenv("TFEXEC_E2ETEST_VERSIONS"); override != "" {
```

Comment:
@kmoe I added this override, and I think we should maybe just run a cron CI run against master instead of on PR, but I'm not sure how to set it up in Circle, so will follow up in a future PR with that.

Reply:
added https://github.com/hashicorp/terraform-exec/issues/70 for this